contrib/neovide/template.py

```diff
@@ -0,0 +1,92 @@
+pkgname = "neovide"
+pkgver = "0.12.1"
+pkgrel = 0
+build_wrksrc = "neovide"
+build_style = "cargo"
+hostmakedepends = [
+    "cargo",
+    "ninja",
+    "libcxx-devel-static",
```

Comment:
??

Reply:
we could also make gn a separate template. they don't actually have breaking changes very often at all (i did that for a while for chromium and it built even with outdated stuff fine, up until rust was added), and all that would be needed is a source mirror (like github)

speaking of the gn.googlesource here is completely unusable because everything from googlesource is from gitiles and there are no stable tarballs (new sha every fetch)

Reply:
Ah that's why the CI failed with mismatching SHA.

Didn't know about that.

Reply:
switched to koji.fedoraproject.org for gn